src/guide/receive_onchain.md

```diff
@@ -24,11 +25,9 @@ let address = swap_info.bitcoin_address;
 
 ```swift
 do {
-  // Optional user-selected dynamic fees (see Connecting to LSP section for details)
-  let opening_fee_params = nil;
-  let request = ReceiveOnchainRequest(opening_fee_params: opening_fee_params);
-
-  let swapInfo = try sdk.receiveOnchain(req: request)
+  let openingFeeParams:OpeningFeeParams? = nil
+  let req = ReceiveOnchainRequest(openingFeeParams: openingFeeParams)
```

Comment:
Can we omit the openingFeeParams from the request? We do have default value in the udl file for that.

Reply:
Updated